components/form/style/index.less

```diff
@@ -105,7 +105,7 @@ input[type='checkbox'] {
       text-align: left;
     }
 
-    label {
+    &>label {
```

Comment:
```suggestion
    &>label {
    > label {
```

Reply:
You don't resolve it.

Reply:
I guess you want to say 'didn't', is it?